src/mapper/github.mapping.ts

```diff
@@ -0,0 +1,64 @@
+import { getReposList, getRepoInfo } from '../client-services/github.client-service'
+
+interface RepoInfo {
+    name: string;
+    description?: string;
+    url: string;
+    languages: string[];
+    language: string;
+    contributors: { 
+        name: string; 
+        avatar_url: string 
+    }[];
+}
+
+async function githubMapping(): Promise<void> {
```

Comment:
1) Export Async function
2) Not sure if this is the right name
    - maybe it should be map<InternalObjectName>s
3) Is this method intended to get detail of a repository or the list of repositories?
4) I think I didn't communicate this well. I think the fetch calls (getReposList, getRepoInfo) calls should be outside this function. We should pass in those information and return a mapped object.
5) Please remove console.logs
6) I think we let the errors escape this layer and a higher layer can catch it or let it go. We should not be consuming the error and not push it up higher
7) I recommend we change this function to the following structure:
    - accept a List of Repos and returns a list of Projects
		- use JS maps:
		
		repos.map(x => {
			return map<InternalObjectName>(x)
		)
	- accept a single Repo and return a singe Project
		- e.g.
		return {
			name: repo.name,
			description: repo.description,
			....
		}

Reply:
@PooriaT , let me know if you want to discuss these.

Reply:
Please also use camel and pascal case only.

e.g.

avatar_url, repo_name and etc...

Reply:
1.  Resolved
2. I am still not sure about naming. What if we decide to use another 3rd party API and want to create another mapper? Isn't it better to use distinctive naming?
3. Yes, this function extracts the name of all repositories as well as their details such as URL, description, contributors, languages, etc. 
4. I have already gotten the repo's name in another function. Those two functions are provided in github.client-serice.ts. I didn't get your point.
5. Resovled
6. I removed